switch_mod/utilities.py

```diff
@@ -223,11 +221,9 @@ def post_solve(model, outputs_dir=None):
     # (the latter occurs when there are problems with licenses, etc)
     
     # replace the old _save_results function
```

Comment:
Ok, I thought that leaving a comment like that would help when people looked for the old function, but if it isn't good practice, then I'll delete it.